config/detekt/detekt-config.yml

```diff
@@ -570,10 +571,11 @@ style:
     active: false
   MaxLineLength:
     active: true
-    maxLineLength: 150
+    maxLineLength: 120
     excludePackageStatements: true
     excludeImportStatements: true
     excludeCommentStatements: false
+    ignoreAnnotated: ['Test']
```

Comment:
Oh nice! So there is a way to do ignore test titles

Reply:
Yep, there are 2 rules: [MaxLineLength](https://detekt.dev/docs/rules/style/#maxlinelength) and [MaximumLineLength](https://detekt.dev/docs/rules/formatting/#maximumlinelength).

The first one checks the line length for code but also comments and everything. This one doesn't have a way of ignoring long method names only, so I disabled it for tests.

The second one only checks the code but also has an `ignoreBackTickedIdentifier` option which is exactly what we need.

So configuring them together like this should cover all the cases we need.